src/components/pages/dashboard/dashboard.js

```diff
@@ -66,38 +72,41 @@ export class Dashboard extends Component {
     // Telemetry stream - START
     const onPendingStart = () => this.setState({ telemetryIsPending: true });
 
-    const telemetry$ = TelemetryService.getTelemetryByDeviceIdP15M()
+    const getTelemetryStream = ({ deviceIds = [] }) => TelemetryService.getTelemetryByDeviceIdP15M(deviceIds)
```

Comment:
Feels like now that there are tiemframes defined as constants somewhere... that it should be a param into the service.

Reply:
The time range values for these apis are different than the ones defines in the helper method in the `methods.js` file.